sdk/identity/azure-identity/inc/azure/identity/workload_identity_credential.hpp

```diff
@@ -61,42 +80,29 @@ namespace Azure { namespace Identity {
     std::string m_tokenFilePath;
 
     explicit WorkloadIdentityCredential(
-        std::string tenantId,
-        std::string const& clientId,
-        std::string const& tokenFilePath,
-        std::string const& authorityHost,
+        std::string const& tenantIdOptions,
```

Comment:
Things like `tenantIdOptions` are not clear at all.
Shouldn't it just be `tenantId`?
`optionalTenantId` maybe? (which we won;t need anymore to be optional, if we initialize struct fields from the environment as I suggested above)